OpenEdXMobile/src/main/java/org/edx/mobile/module/prefs/LoginPrefs.java

```diff
@@ -90,6 +93,17 @@ public void setSubtitleLanguage(@Nullable String language) {
         pref.put(PrefManager.Key.TRANSCRIPT_LANGUAGE, language);
     }
 
+    /**
+     * @return Playback speed
+     */
+    public float getDefaultPlaybackSpeed() {
```

Comment:
@farhan We use `getDefaultPlaybackSpeed` because method is used to retrieve the **default** value for the playback speed when user come again on the video screen.

Reply:
I got it how are you making its sense.
But IMO that's not default value, that will be user selected, our default value will always be 'Normal' when user would have never selected it from menu (what real sense of default sounds to me in code).